src/easy/handler.rs

```diff
@@ -3698,6 +3723,20 @@ fn double_seconds_to_duration_sub_second2() {
     assert_eq!(dur.subsec_nanos(), 500_000_000);
 }
 
+impl CURLPtr {
+    fn as_ptr(&self) -> *mut curl_sys::CURL {
+        self.0
+    }
+}
```

Comment:
Since the struct is package-private, perhaps this is too much ceremony and we should just reach into it with `.0` everywhere. I don't think there's a perf difference, mostly a matter of taste. Happy to remove this.

Reply:
Or we could `impl Deref` - maybe that would be nicer?